src/components/accordion.jsx

```diff
@@ -0,0 +1,243 @@
+})(MuiAccordion);
+
+const AccordionSummary = withStyles({
+  root: {
+    backgroundColor: "#f7f7f7",
+    marginBottom: -1,
+    minHeight: 56,
+    height: 61,
+
+    "&$expanded": {
+      minHeight: 56,
+      outline: "none",
+      border: "none",
+    },
+  },
+
+  content: {
+    "&$expanded": {
+      margin: "14px 0",
+    },
+  },
+  expanded: {},
+})(MuiAccordionSummary);
+
+const AccordionDetails = withStyles((theme) => ({
+  root: {
+    backgroundColor: "#f7f7f7",
+    padding: theme.spacing(2),
+    paddingTop: theme.spacing(0),
+  },
+}))(MuiAccordionDetails);
+
+export default function CustomizedAccordions() {
+  const [expanded, setExpanded] = useState("panel1");
+
+  const handleChange = (panel) => (event, newExpanded) => {
+    setExpanded(panel);
+  };
+
+  const handleHeaderClassChange = () => {
+    const arrowElementName = `acc-${expanded}-arrow`;
+    console.log(arrowElementName);
+    const arrowElement = document.getElementById(arrowElementName);
+    arrowElement.classList.toggle("accordion-arrow-expanded");
+
+    const headerElementName = `acc-${expanded}-header`;
+    console.log(headerElementName);
+    const headerElement = document.getElementById(headerElementName);
+    headerElement.classList.toggle("accordion-header-expanded");
+  };
+
+  useEffect(() => {
+    console.log(expanded);
+    const arrowElementName = `acc-${expanded}-arrow`;
+    console.log(arrowElementName);
+    const arrowElement = document.getElementById(arrowElementName);
+    arrowElement.classList.toggle("accordion-arrow-expanded");
+
+    const headerElementName = `acc-${expanded}-header`;
+    console.log(headerElementName);
+    const headerElement = document.getElementById(headerElementName);
+    headerElement.classList.toggle("accordion-header-expanded");
+
+    return () => {
+      arrowElement.classList.toggle("accordion-arrow-expanded");
+      headerElement.classList.toggle("accordion-header-expanded");
+    };
+  });
+
+  return (
+    <div className="accordion">
+      <Accordion
+        square
+        expanded={expanded === "panel1"}
+        onChange={handleChange("panel1")}
+      >
+        <AccordionSummary aria-controls="panel1d-content" id="panel1d-header">
+          <div id="acc-panel1-header" className="accordion-head">
+            First Common Question
+            <span id="acc-panel1-arrow" className="accordion-arrow"></span>
+          </div>
+        </AccordionSummary>
+        <AccordionDetails>
+          <div className="accordion-details">
+            <p>
+              Duis vulputate porttitor urna sit amet pretium. Phasellus sed
+              pulvinar eros, condimentum consequat ex. Suspendisse potenti.
+            </p>
+            <p>
+              Pellentesque maximus lorem sed elit imperdiet mattis. Duis posuere
+              mauris ut eros rutrum sodales. Aliquam erat volutpat.
+            </p>
+          </div>
+        </AccordionDetails>
+      </Accordion>
+      <Accordion
```

Comment:
if the layout is the same for every accordion item, I would put the data into an array of objects and iterate it. So IU don't have to copy and paste everything.

Reply:
fixed...

Reply:
And as per your suggestion, I updated the services carousel part too with map function.